src/odinapi/templates/dataaccess.html

```diff
@@ -377,19 +377,76 @@ <h2>Recommended projects</h2>
 <p>
 Note that all spaces in the product names need to be replaced by
 <code>%20</code> in the request.
+</p>
+<p>
+For a detailed discussion of the different products, see the
+<a href="{{ url_for('static', filename='documents/PVER.pdf') }}">Odin/SMR Product Validation and Evolution Report</a>.
 </p>
           <div>
             <table class="table">
               <tbody>
                 <tr>
-                  <td><b>Species:</b></td>
-                  <td><b>Project:</b></td>
-                  <td><b>Product name:</b></td>
+                  <td><b>Species</b></td>
```

Comment:
actually there's something called table header and it would be more suitable than manual bolding here.